File: Mobile/D2/LambdaPP_AndroidApp_D2/app/src/main/java/com/example/lambdapp_androidapp_d2/models/Employee.java
package com.example.lambdapp_androidapp_d2.models;

import android.media.Image;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;
import java.sql.Date;
import java.util.List;

public class Employee {
    public int emp_ID;
    public String emp_Username;
    public String emp_Password;
    public String emp_Name;
    public String emp_Sur;
    public Boolean emp_IsAdmin = false;
    public Boolean emp_IsContractor = false;
    // public Image? Emp_ID_Image;
    public byte[] emp_ID_Image;
    public String emp_DateRegistered;
    public Boolean emp_Enabled = true;
    public List<Task_Employee_Bridge> task_employee_bridges;
    public List<Task> tasks;
    //note: these are the ratings they have GIVEN, not received
    public List<Rating> ratingsGiven;
    public List<Employee_Skill_Bridge> employee_Skill_Bridges;
    public List<Skill> skills;
    public List<Task_Update> task_Updates;
    public double emp_Rating;

}

File: Dev/WebApp/LambdaPP_ProjectCore_Alpha/LamdbdaPP_WebAPI_Alpha/Database/Models/Employee.cs
using Humanizer;
using LamdbdaPP_WebAPI_D3.Database.Models;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Drawing;
using System.Text.Json.Serialization;

namespace LambdaPP_WebAPI_Alpha.Database.Models
{
    //This is a model of the "employee" table. It has to match up exactly with the data in the database,
    //in order for the DbContext to work properly!
    //This is what will be returned by the DBContext when it accesses the database.
    //Will need to make models for all other tables we want to access - Gage
    public class Employee
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [JsonPropertyName("emp_ID")]
        public int? Emp_ID { get; set; }
        [JsonPropertyName("emp_Username")]
        public string? Emp_Username { get; set; }
        [JsonPropertyName("emp_Password")]
        public string? Emp_Password { get; set; }
        [JsonPropertyName("emp_Name")]
        public string? Emp_Name { get; set; }
        [JsonPropertyName("emp_Sur")]
        public string? Emp_Sur { get; set; }
        [JsonPropertyName("emp_IsAdmin")]
        public bool Emp_IsAdmin { get; set; }
        [JsonPropertyName("emp_IsContractor")]
        public bool Emp_IsContractor { get; set; }
        // public Image? Emp_ID_Image { get; set; }
        [JsonPropertyName("emp_ID_Image")]
        public byte[]? Emp_ID_Image { get; set; }
        [JsonPropertyName("emp_DateRegistered")]
        public DateTime Emp_DateRegistered { get; set; }
        [JsonPropertyName("emp_Enabled")]
        public bool Emp_Enabled { get; set; }
        [JsonPropertyName("task_Employee_Bridges")]
        public List<Task_Employee_Bridge>? Task_Employee_Bridges { get; set; }
        [JsonPropertyName("tasks")]
        public List<Task>? Tasks { get; set; }
        //note: these are the ratings they have GIVEN, not recieved
        [JsonPropertyName("ratingsGiven")]
        public List<Rating>? ratingsGiven { get; set; }

        [JsonPropertyName("employee_Skill_Bridges")]
        public List<Employee_Skill_Bridge>? Employee_Skill_Bridges { get; set; }
        [JsonPropertyName("skills")]
        public List<Skill>? skills { get; set; }

        [JsonPropertyName("task_Updates")]
        public List<Task_Update>? Task_Updates { get; set; }


        [JsonPropertyName("emp_Rating")]
        public decimal Emp_Rating { get; set; }

    }
}
